lips/lip-1.mdx

```diff
@@ -11,156 +11,136 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 ### On-Chain Account Setup
+
 A VASP's on-chain account must be created and set up in accordance with the standard VASP account creation process.
 
-Of particular note for off-chain APIs is the compliance key value.  This is the Ed25519 key with which a VASP signs travel rule statements on-chain and the required information within the KycDataObject exchanged via the Off-Chain Protocol.  During account creation, a VASP will set their compliance key which will be stored under the primary VASP account.
+Of particular note for off-chain APIs is the compliance key and base url values. The compliance key value is the Ed25519 key with which a VASP signs travel rule statements which are verified on-chain and all off-chain requests and responses. During account creation, a VASP will set their compliance key and base url which will be stored under the primary VASP account (Parent VASP account).
 
 ### HTTP end-point
 
-Each VASP exposes an HTTPS POST end point at `https://<hostname>:<port>/<protocol_version>/<LocalVASPAddress>/<RemoteVASPAddress>/command`.
+Each VASP exposes an HTTPS POST end point at `https://<hostname>:<port>/<protocol_version>/command`. The `protocol_version` is `v2` for this iteration of the off-chain APIs.
+The base url value should be the url without path `/<protocol_version>/command`, e.g. `https://<hostname>:<port>`.
+
+### HTTP Headers
 
-* `protocol_version` is initially 0 for this iteration of the off-chain APIs.
-* `LocalVASPAddress` and `RemoteVASPAddress` are the bech32 VASP addresses of the respective VASPs - with the subaddress set to 0's as per [LIP 5](https://lip.libra.org/lip-5/).
-    * An example URL: http://127.0.0.1:7091/v1/lbr1pg9q5zs2pg9q5zs2pg9q5zs2pgyqqqqqqqqqqqqqqspa3m/lbr1pgfpyysjzgfpyysjzgfpyysjzggqqqqqqqqqqqqqxjt54z/command/
+All HTTP requests must contain:
 
+* A header `X-REQUEST-ID` with a unique UUID (according to [RFC4122](https://tools.ietf.org/html/rfc4122)) for the request, used for tracking requests and debugging. Responses must have the same string in the `X-REQUEST-ID` header value as the requests they correspond to.
+* A header `X-REQUEST-SENDER-ADDRESS` with the HTTP request sender's VASP DIP-5 address used in the command object. The HTTP request sender must use the compliance key of the VASP account linked with this address to sign the request JWS body, and the request receiver uses this address to find the request sender's compliance key for verifying the JWS signature. For example: if the http request sender is the PaymentCommand receiver actor, then this value should be the receiver actor address from PaymentObject in PaymentCommand, and if the receiver actor address contains a ChildVASP onchain account, the request JWS body must be signed by the compliance key stored in the ParentVASP account of the ChildVASP account.
```

Comment:
I'm trying to make this more clear: ignore me if you don't think it's better:

```suggestion
* A header `X-REQUEST-SENDER-ADDRESS` with the HTTP request sender's VASP DIP-5 address used in the command object. The HTTP request sender must use the compliance key of the VASP account linked with this address to sign the request JWS body, and the request receiver uses this address to find the request sender's compliance key to verify the JWS signature. For example: VASP A transfers funds to VASP B. In the HTTP request A sends to B, it contains `X-REQUEST-SENDER-ADDRESS` as VASP A's address. In HTTP request B sends to A, it contains `X-REQUEST-SENDER-ADDRESS` as VAPS B's address. 
```

Reply:
Should link "DIP-5 address" with the DIP spec

Reply:
"In HTTP request B sends to A,..." -> "An HTTP request B sends to A should contain VASP B's address as `X-REQUEST-SENDER-ADDRESS`.

Reply:
Updated